index.js

```diff
@@ -24,6 +24,12 @@ program
       }
     }
 
+    var num = numberOfResults;
+
+    if(numberOfResults === undefined) {
+      num = 3;
+    }
```

Comment:
Can we consolidate this to a ternary statement instead? I think this can be combined to one line.

Reply:
We can probably achieve that by setting the undefined check to a variable

Reply:
i think we can remove this if i just add the default value for it right?

Reply:
I believe we can actually @psyduc